src/Form/CitizenProjectImageType.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace AppBundle\Form;
+
+use AppBundle\MediaGenerator\Command\CitizenProjectImageCommand;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class CitizenProjectImageType extends CitizenProjectMediaType
```

Comment:
Could you explain why form type inheritance mecanism is not used ? (`getParent` method)

Reply:
for avoiding duplicate `getBlockPrefix` and `getParent` methods in children classes